src/pages/Browse/index.spec.tsx

```diff
@@ -74,32 +92,55 @@ describe(__filename, () => {
     );
   });
 
-  it('calls the API to load the version info on mount', async () => {
-    const addonId = 12345;
-    const version = {
-      ...fakeVersion,
-      id: 999,
-    };
+  it('renders the content of the default file when a version has been loaded', () => {
+    const version = fakeVersion;
+
+    const store = configureStore();
+    store.dispatch(versionActions.loadVersionInfo({ version }));
+
+    const root = render({ store, versionId: String(version.id) });
+
+    expect(root.find(Highlight)).toHaveLength(1);
+    expect(root.find(Highlight)).toHaveProp('className', 'auto');
```

Comment:
This is just asserting a static prop, which I think we don't normally do.

Reply:
This one is important because it configures the syntax HL feature.